x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -73,6 +73,7 @@ export const UNISOLATE_HOST_ROUTE = `${BASE_ENDPOINT_ROUTE}/unisolate`;
 
 /** Base Actions route. Used to get a list of all actions and is root to other action related routes */
 export const BASE_ENDPOINT_ACTION_ROUTE = `${BASE_ENDPOINT_ROUTE}/action`;
+export const BASE_ENDPOINT_ACTION_ALERTS_ROUTE = `${BASE_ENDPOINT_ROUTE}/alerts`;
```

Comment:
can you explain what this route is?

I noticed the name of the `const`  `_ALERTS_`, but the route definition is using `BASE_ENDPOINT_ROUTE` rather than `BASE_ENDPOINT_ACTION_ROUTE`

Reply:
Yes, exactly. This is a route meant to return back `actions`, so IMO, it should be mounted under `/action/`